### PR TITLE
Update template files

### PR DESCRIPTION
**Pull request by Normo**
This PR updates the RSD environment and docker-compose template files by adding missing environment variables.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -34,13 +34,18 @@ rsd_container_registry_path: "ghcr.io/research-software-directory/rsd-saas"
 
 # RSD ENV variables
 rsd_domain: "localhost"
-rsd_ssl_domains: "localhost"
+rsd_auth_providers: "SURFCONEXT;HELMHOLTZAAI"
+rsd_hgfaai_client_id: "rsd-dev"
+rsd_hgfaai_client_secret: "changeme"
+rsd_hgfaai_well_known_url: "https://login-dev.helmholtz.de/oauth2/.well-known/openid-configuration"
+rsd_hgfaai_token_url: "https://login-dev.helmholtz.de/oauth2/token"
+rsd_hgfaai_allow_external_users: "https://login-dev.helmholtz.de/oauth2/token"
```
Comment (tobiashuste) [inline defaults/main.yml:42]: This should probably be `true` or `false`?

File: templates/rsd-secrets.env.j2 [hunk 2]
```diff
@@ -3,68 +3,115 @@
...
+# consumed by: authentication, frontend/utils/loginHelpers
+NEXT_PUBLIC_HELMHOLTZAAI_SCOPES=openid+profile+email+eduperson_principal_name
+# consumed by: frontend/utils/loginHelpers
+NEXT_PUBLIC_HELMHOLTZAAI_RESPONSE_MODE=query
+# consumed by services: authentication
+AUTH_HELMHOLTZAAI_TOKEN_URL={{ rsd_hgfaai_token_url }}
+{% if rsd_hgfaai_allow_external_users | length > 0 %}
+# consumed by: authentication
+# uncomment if you want to allow users from non-Helmholtz centres or social IdPs:
+HELMHOLTZAAI_ALLOW_EXTERNAL_USERS={{ rsd_hgfaai_allow_external_users | bool }}
```
Comment (tobiashuste) [inline templates/rsd-secrets.env.j2:75]: This might be the reason for the failing CI job?

File: defaults/main.yml (hunk 1 above)
Comment (tobiashuste) [reply defaults/main.yml:42]: The variable is also missing in the README I think.

**Review by tobiashuste — CHANGES_REQUESTED** [review]
Thanks, @Normo. I added a couple of thoughts also related to the failing CI pipeline.

File: templates/rsd-secrets.env.j2 (hunk 2 above)
Comment (Normo) [reply templates/rsd-secrets.env.j2]: This should be fixed now.

**Review by tobiashuste — APPROVED** [review]
This looks reasonable now and the pipeline succeeds. Thanks, @Normo :rocket:

**Pull request merged**